### ISSUE TITLE
getBondOrdersForAtom for GroupAtom is ambiguous

### ISSUE THREAD

**Issue by goldmanm** [ReactionMechanismGenerator/RMG-Py#865]
The `getBondOrdersForAtom` method doesn't seem to work for GroupAtom class in group.py. It doesn't seem to treat `bond.order` as a list. I am not sure what the indented output of the function is. To me, it would seem to be a list of potential bond orders, based on all the options of each GroupBond object, but the method seems to output only one number. The code is below, for convenience:

```
    def getBondOrdersForAtom(self):
        """
        This helper function is to help calculate total bond orders for an
        input atom.
        Some special consideration for the order `B` bond. For atoms having
        three `B` bonds, the order for each is 4/3.0, while for atoms having other
        than three `B` bonds, the order for  each is 3/2.0
        """
        num_B_bond = 0
        order = 0
        for _, bond in self.bonds.iteritems():
            if bond.order == 'B':
                num_B_bond += 1
            else:
                order += mol.bond_orders[bond.order]
        if num_B_bond == 3:
            order += num_B_bond * 4/3.0
        else:
            order += num_B_bond * 3/2.0
return order

```

**Comment by nyee** [comment]
Background: This function is supposed to be used to help calculate valency for atoms. For example, we could use it for the updateLonePair to get the number of electrons tied up in bonds then calculate the number of lone pairs after the fact. We specifically made it a function because several places in the code had "if" statement blocks that do the same thing, but we wanted to change the way we handle B bonds, so it made sense to centralize all these blocks to one statement. 

So I'm pretty sure this was mostly intended just for `Atom` and not for `GroupAtom`. I'm also not entirely sure what it should do in the case of wildcard bonds. Maybe it just ignore wildcards? I don't think this function is used for anything in `GroupAtom`, right?

**Comment by goldmanm** [comment]
I can't find a place this method in `GroupAtom` is called, but the version in `Atom` class is fully functional. Even though this method isn't functional, with the change to `GroupBond` it is even farther away from functional. 

I'm going to remove it in the branch `bond_doubles`, if that is alright.

**Comment by goldmanm** [comment]
solved with #864
